wpilibc/src/test/native/cpp/simulation/ADIS16488_IMUSimTest.cpp

```diff
@@ -43,12 +43,6 @@ TEST(ADIS16448_IMUSimTest, Offset) {
   gyro.Reset(OFFSET);
   EXPECT_EQ(OFFSET, gyro.GetRotation3d());
 
-  sim.SetGyroAngleZ(10_deg);
-  EXPECT_EQ(100_deg, gyro.GetAngle());
-
-  sim.SetGyroAngleZ(90_deg);
-  EXPECT_EQ(180_deg, gyro.GetAngle());
```

Comment:
Why are these tests removed?

Reply:
They are being replaced with a `sim.SetGyroOrientation(Rotation3d)` function and associated test cases because the IMUs use a quaternion internally and so it makes more sense for sim to work that way. Users can still set angles via the Roll/Pitch/Yaw Rotation3d constructor, so the usability difference is not significant IMO.